MidtransKit/MidtransKit/VTAddCardController.m

```diff
@@ -340,13 +340,13 @@ - (void)formatter_didTextFieldChange:(MidtransUICardFormatter *)formatter {
     if (self.view.cardNumber.text.length < 1) {
         self.view.cardFrontView.numberLabel.text = @"XXXX XXXX XXXX XXXX";
         self.view.cardFrontView.iconView.image = nil;
-        self.view.cardNumber.infoIcon = nil;
+        self.view.cardNumber.info1Icon = nil;
     }
     else {
         self.view.cardFrontView.numberLabel.text = self.view.cardNumber.text;
         NSString *originNumber = [self.view.cardNumber.text stringByReplacingOccurrencesOfString:@" " withString:@""];
-        self.view.cardNumber.infoIcon = [self.view iconDarkWithNumber:originNumber];
-        self.view.cardNumber.infoBankIcon = [self.view iconWithBankName:self.filteredBinObject.bank];
+        self.view.cardNumber.info1Icon = [self.view iconDarkWithNumber:originNumber];
```

Comment:
my purpose to use info(x)Icon is to make it more general, because this textfield not only used for credit card